src/components/React/ColorMappingController.js

```diff
@@ -1,73 +1,31 @@
 import React, { Component } from "react";
 import ReactModal from "react-modal";
+// import { DataTable } from "primereact/datatable";
+// import { Column } from "primereact/column";
+import "primereact/resources/themes/nova/theme.css";
+import "primereact/resources/primereact.min.css";
+import "primeicons/primeicons.css";
+
 import { connect } from "react-redux";
```

Comment:
how big of a lift is it to transition this component to a functional one at this point?

Reply:
Definitely a separate PR but could conceivably be done in a day or two. Added issue #38